SDK v2 migration guide.md

```diff
@@ -0,0 +1,399 @@
+| JobClient#scheduleDeviceMethod(String, String, Long, Long, Object, Date, long); | ScheduledJobsClient#scheduleDirectMethod(String, String, String, Date, DirectMethodsJobOptions);  |
+| JobClient#getJob(String); | ScheduledJobsClient#getJob(String);  |
+| JobClient#cancelJob(String); | ScheduledJobsClient#cancelJob(String);  |
+| JobClient#queryDeviceJob(String); | ScheduledJobsClient#query(String), QueryClient#queryJobs(String);  |
+
+
+#### ServiceClient
+
+| V1 class#method  | Equivalent V2 class#method |
+|:---|:---|
+| ServiceClient#open(); | MessagingClient#open();  |
+| ServiceClient#close(); | MessagingClient#close();  |
+| ServiceClient#send(String, Message); | MessagingClient#send(String, Message);  |
+| ServiceClient#send(String, String, Message); | MessagingClient#send(String, String, Message);  |
+| ServiceClient#getFileUploadNotificationReceiver(); | new FileUploadNotificationProcessorClient(String, IotHubServiceClientProtocol, Function<FeedbackBatch, AcknowledgementType>);  |
+| FileUploadNotificationReceiver#open(); | FileUploadNotificationProcessorClient#start();  |
+| FileUploadNotificationReceiver#close(); | FileUploadNotificationProcessorClient#stop();  |
+| FileUploadNotificationReceiver#receive(); | no equivalent method**  |
+| FileUploadNotificationReceiver#receive(int); | no equivalent method**  |
+| ServiceClient#getFeedbackReceiver(); | new MessageFeedbackProcessorClient(String, IotHubServiceClientProtocol, Function<FileUploadNotification, AcknowledgementType>);  |
+| FeedbackReceiver#open(); | MessageFeedbackProcessorClient#start();  |
+| FeedbackReceiver#close(); | MessageFeedbackProcessorClient#stop();  |
+| FeedbackReceiver#receive(); | no equivalent method**  |
+| FeedbackReceiver#receive(int); | no equivalent method**  |
+
+** Your FileUploadNotificationProcessorClient and MessageFeedbackProcessorClient will start receiving as soon as the client is started
+
+
+#### Other notable breaking changes
+
+- Trust certificates are read from the physical device's trusted root certification authorities certificate store rather than from source.
+  - Users are expected to install the required public certificates into this certificate store if they are not present already
+  - See [this document](./upcoming_certificate_changes_readme.md) for additional context on which certificates need to be installed.
+- The Bouncycastle dependencies have been removed
+  - The Bouncycastle dependencies were used for some certificate parsing logic that has been removed from the SDK.
+- Reduced access levels to classes and methods that were never intended to be public where possible 
+- Removed service error code descriptions that the service would never return the error code for
+- Reduce default SAS token time to live from 1 year to 1 hour for security purposes
+- Removed unnecessary synchronization on service client APIs to allow for a single client to make multiple service APIs simultaneously
```

Comment:
...make multiple service ~APIs~ _API calls_ simultaneously. ?

Reply:
Q - if we say "multiple", do you think people will ask to quantify that; i.e. how many calls can we have in parallel?
How about if we say "...to allow for a single client to make service API calls simultaneously."?

Reply:
Or if we can actually quantify and state it, even better :)

Reply:
We allow infinitely many calls in parallel, so I like your suggestion of  "...to allow for a single client to make service API calls simultaneously." here